geode/advanced-examples/cascading-functions/pom.xml

```diff
@@ -0,0 +1,62 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <parent>
+        <artifactId>advanced-examples</artifactId>
+        <groupId>example.springdata.geode</groupId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>cascading-functions</artifactId>
```

Comment:
Almost every pom file contains an excessive dependency configuration with several additional dependencies exclusions. It would make sense to move the initial dependency setup into `geode/pom.xml` and adjust the pom only for the modules where it's required to minimize the cognitive load per example.

Reply:
Moved common dependencies to parent pom.